docs/source/getting-started/model-registry/step2-load-registered-model.rst

```diff
@@ -0,0 +1,76 @@
+Registering a Model
+===================
+
+Now that we have registered a model, there are many ways to bring that model into memory. Typically,
+users reference a model by its name, which is a unique identifier determined when the model is 
+logged. However, you can also get the model via absolute paths to the artifact, run IDs combined 
```

Comment:
Instead of the non-exhaustive list of how to get a reference to the serialized model, could you list out, with bullet points, a few ways of getting a reference (from the returned value from logging a model, from the run_id + artifact uri, the model_uri, the saved path)?

Reply:
Nevermind. You already list them down below. Could you just move that section up here? ;)